TestCases.md

```diff
@@ -0,0 +1,487 @@
+9. Verify that the transaction ID is truncated correctly.
+10. Ensure the transaction type is correctly displayed as either "spend" or "spawn."
+11. Verify that the date is displayed correctly.
+12. Ensure the layer and epoch information is displayed correctly next to the date, and the values are correct.
+13. Ensure the amount is formatted appropriately with the correct number of decimal places.
+14. Ensure that the transaction cost is displayed correctly.
+15. Verify that the tab displays an appropriate message when there are no transactions to show.
+16. Verify that the transactions' history loads quickly and correctly, even with a large number of transactions.
+17. Ensure smooth scrolling through the list of transactions, especially with a large dataset.
+
+ **Details**
+
+1. Verify that clicking on a transaction row opens a sidebar showing detailed information about the transaction.
+2. Ensure the sidebar loads correctly and displays all the relevant information, correct for the given transaction.
+3. Ensure the full hexadecimal transaction ID is displayed correctly.
+4. Check that the transaction status is displayed with the correct text (e.g., Pending, Confirmed).
+5. Ensure the status color code corresponds correctly (e.g., orange for processing, green for confirmed).
+6. Verify that the sender's address is displayed correctly.
+7. Verify that the recipient's address is displayed correctly.
+8. Ensure the nonce value is displayed correctly.
+9. Verify that there are clickable explorer links next to the transaction ID, sender address, recipient address, layer, epoch, and one general.
+10. Check that clicking the explorer link opens the transaction details in the explorer. (all buttons work, each button leads to the given detail page)
+11. Verify that there is a copy button next to the transaction ID, sender, and recipient addresses.
+12. Verify that clicking the copy button copies the given detail to the clipboard.
+13. Check for a confirmation message or indication after copying
+14. Check behavior when network issues occur while loading the transaction details.
+15. Ensure the sidebar layout is clear and information is easy to read.
+
+## Rewards History Tab
+
+1. Ensure the Rewards History tab is visible and clickable.
+2. Verify that the Rewards History tab loads successfully.
+3. Check that the rewards history shows correct and detailed information for each reward.
+4. Ensure that each reward entry includes details such as Date and time in the format: YYYY-MM-DD HH, Layer and Epoch information, Reward amount, Reward for layer amount, Reward for fees amount. Explorer link/button.
+5. Verify that the layer and epoch information are displayed correctly next to the correct date and time.
+6. Ensure that the reward amounts are displayed correctly and formatted appropriately with the correct value and number of decimal places.
+7. Check that clicking the explorer link/button opens the reward details in the explorer.
```

Comment:
Did it work for you? (The wallet I checked has only rewards from 2023 and it didn't work)